agent/stats/engine.go

```diff
@@ -66,7 +66,7 @@ type DockerContainerMetadataResolver struct {
 // defined to make testing easier.
 type Engine interface {
 	GetInstanceMetrics() (*ecstcs.MetricsMetadata, []*ecstcs.TaskMetric, error)
-	ContainerDockerStats(taskARN string, containerID string) (*types.StatsJSON, error)
+	ContainerDockerStats(taskARN string, containerID string) (*types.StatsJSON, NetworkStatsPerSec, error)
```

Comment:
Can this be `*NetworkStatsPerSec` instead of `NetworkStatsPerSec`, this way nil value can be returned instead of `NetworkStatsPerSec{}` of when required.

Reply:
Changed it.